components/property_info.rst

```diff
@@ -0,0 +1,432 @@
+.. index::
+    single: PropertyInfo
+    single: Components; PropertyInfo
+
+The PropertyInfo Component
+==========================
+
+The PropertyInfo component provides the functionality to get information about class properties using metadata of
+popular sources.
+
+Whilst the :doc:`PropertyAccess component</components/property_access/introduction>` allows you to read and write values
```

Comment:
I see multiple occurrences of "whilst".

See http://dictionary.cambridge.org/grammar/british-grammar/while-and-whilst